docs/concepts/hybrid-custody/guides/account-model.mdx

```diff
@@ -0,0 +1,334 @@
+
+We’ve leveraged this feature in a (proposed) standard so that dapps can implement a hybrid custody model whereby the dapp creates an account it controls, then later delegates access on that account to the user once they’ve authenticate with their wallet. All related constructs are used together in the [`HybridCustody` contract](https://testnet.contractbrowser.com/A.96b15ff6dfde11fe.HybridCustody) to define the standard. The delegated access on child accounts is stored and mediated by the parent account's `Manager` which maintains Capabilities on `ProxyAccount` and `ChildAccount` resources, themselves defining restricted and unrestricted access respectively.
+
+Therefore, the presence of a `Manager` in an account implies there are potentially associated accounts for which the owning account has delegated access. This resource is intended to be configured with a pubic Capability enabling querying of an account's child account addresses via `getAccountAddresses()` and `getOwnedAccountAddresses()`. As you can deduce from these two methods, there is a notion of "owned" accounts which we'll expand on in a bit.
+
+A wallet or marketplace wishing to discover all of a user’s accounts and assets within them can do so by first looking to the user’s `Manager`.
+
+## Identifying Account Hierarchy
+
+To clarify, insofar as the standard is concerned, an account is a parent account if it contains a `Manager` resource, and an account is a child account if it contains at minimum a `ChildAccount` or additionally a `ProxyAccount` resource.
+
+We can see that the user’s `Manager.accounts` point to the address of its child account in its index, with corresponding Capabilities giving the Manager access to those accounts. Likewise, the child account’s `ProxyAccount.parentAddress` points to the user’s account as its parent address. This makes it easy to both identify whether an account is a parent, child, or both, and its associated parent/child account(s). `ChildAccount` resources underly all account delegations, so can have multiple parents whereas `ProxyAccount`s are 1:1. This provides more granular revocation as each parent account has its own Capability path on which its access relies.
+
+### Restricted vs. Owned Accounts
+
+It's worth noting here that `ProxyAccount` Capabilities enable access to the underlying account according to rules configured by the delegator. The `ProxyAccount` maintains these rules along with a `ChildAccount` resource within which the AuthAccount Capability is stored. Anyone with access to the surface level `ProxyAccount` can then access the underlying AuthAccount, but only according the the pre-defined rule set. These rules are fundamentally a list of Types that can/cannot be retrieved from an account.
```

Comment:
```suggestion
It's worth noting here that `ProxyAccount` Capabilities enable access to the underlying account according to rules configured by the child account who is delegating the authority.
The `ProxyAccount` and `ChildAccount` resources maintain these rules because they are where the `AuthAccount` Capabilities are stored. 
Anyone with access to the surface level `ProxyAccount` can then access the underlying AuthAccount, but only according the the pre-defined rule set. These rules are fundamentally a list of Types that can/cannot be retrieved from an account.
```